pkgs/misc/lguf-brightness/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, cmake, libusb, ncurses }:
+
+stdenv.mkDerivation rec {
+  name = "lguf-brightness";
+
+  version = "d194272b";
+
+  src = fetchFromGitHub {
+    owner = "periklis";
+    repo = "${name}";
+    rev = "d194272b7a0374b27f036cbc1a9be7f231d40cbb";
+    sha256 = "0zj81bqchms9m7rik1jxp6zylh9dxqzr7krlj9947v0phr4qgah4";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  buildInputs = [ libusb ncurses ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    mv lguf_brightness $out/bin
```

Comment:
Can be changed to `install -D lguf_brightness $out/bin/lguf_brightness`

That will also create the directory for you